test/clj_gcp/integration/pub_sub/core_test.clj

```diff
@@ -1,6 +1,6 @@
 (ns clj-gcp.integration.pub-sub.core-test
   (:require [cheshire.core :as json]
-            [clj-gcp.integration.pub-sub.utils :as mqu]
```

Comment:
Does `clj-gcp.integration.pub-sub.utils` still exist?

Reply:
Yup, deleted it now.